internal/caravan/tradeagreement.lua

```diff
@@ -1,23 +1,83 @@
 --@ module = true
-
+local dlg = require('gui.dialogs')
 local gui = require('gui')
 local overlay = require('plugins.overlay')
 local widgets = require('gui.widgets')
 
 TradeAgreementOverlay = defclass(TradeAgreementOverlay, overlay.OverlayWidget)
-TradeAgreementOverlay.ATTRS{
-    desc='Adds select all/none functionality when requesting trade agreement items.',
-    default_pos={x=45, y=-6},
-    default_enabled=true,
-    viewscreens='dwarfmode/Diplomacy/Requests',
-    frame={w=25, h=3},
-    frame_style=gui.MEDIUM_FRAME,
-    frame_background=gui.CLEAR_PEN,
+TradeAgreementOverlay.ATTRS {
+    desc = 'Adds select all/none functionality when requesting trade agreement items.',
+    default_pos = { x = 45, y = -6 },
+    default_enabled = true,
+    viewscreens = 'dwarfmode/Diplomacy/Requests',
+    frame = { w = 40, h = 4 },
+    frame_style = gui.MEDIUM_FRAME,
+    frame_background = gui.CLEAR_PEN,
 }
```

Comment:
Inside of tables, the convention (at least inside of DFHack code) is to have no spaces around the `=`